```
kubernetes-upgrade-gke-1.1-1.2-step6-e2e-old/20
Traceback (most recent call last):
  File "gen_json.py", line 187, in <module>
    tests = get_tests(server, prefix)
  File "gen_json.py", line 167, in get_tests
    for name, duration, failed, skipped in get_tests_from_build(job, build):
  File "gen_json.py", line 143, in get_tests_from_build
    for test in get_tests_from_junit_path(junit_path):
  File "gen_json.py", line 135, in get_tests_from_junit_path
    for test in gcs_get_tests(path):
  File "gen_json.py", line 117, in gcs_get_tests
    root = ET.fromstring(data)
  File "/usr/lib/python2.7/xml/etree/ElementTree.py", line 1301, in XML
    parser.feed(text)
  File "/usr/lib/python2.7/xml/etree/ElementTree.py", line 1641, in feed
    self._parser.Parse(data, 0)
UnicodeEncodeError: 'ascii' codec can't encode character u'\u0142' in position 1341: ordinal not in range(128)
```

--- a/hack/jenkins/test-history/gen_json.py
+++ b/hack/jenkins/test-history/gen_json.py
@@ -111,7 +111,6 @@ def gcs_get_tests(path):
     except zlib.error:
         # Don't fail if it's not gzipped.
         pass
-    data = data.decode('utf-8')
 
     try:
         root = ET.fromstring(data)

